cfme/v2v/migrations.py

```diff
@@ -815,6 +832,17 @@ def step(self):
         self.prerequisite_view.create_infrastructure_mapping.click()
 
 
+@navigator.register(InfrastructureMapping, 'Edit')
+class EditInfrastructureMapping(CFMENavigateStep):
+    VIEW = EditInfrastructureMappingView
+
+    prerequisite = NavigateToAttribute('parent', 'All')
+
+    def step(self):
+        self.prerequisite_view.infra_mapping_list.edit_mapping(
```

Comment:
I think there needs to be a check here on the return of `edit_mapping`, since if its `False` we have failed to navigate and shouldn't leave it to timeout on wait_displayed.

I think you should raise `ItemNotFound` on a `False` return from edit_mapping.